### ISSUE TITLE
Feature Request - Update Nuget targets to link unmanaged resources

### ISSUE THREAD

**Issue by amaitland** [cefsharp/CefSharp#2156]
https://stackoverflow.com/a/30316946/4583726
https://msdn.microsoft.com/en-us/library/0c6xyb66(VS.80).aspx

Above link suggests adding unmanaged files using a none target and setting a link, should improve support for for creating library projects that reference CefSharp. 

Just adding as reference in case anyone wants to take up the challenge.

**Comment by perlun** [comment]
Thanks for the suggestion @amaitland! I hope someone has the time to investigate this sometime, it would be a nice addition for certain use cases.

**Comment by amaitland** [comment]
`cef.redist` packages were restructed to as part of this change, the `.targets` files were renamed to `.props`, see https://github.com/cefsharp/cef-binary/commit/9eef26e5d913e02284d4f8522fce1ada22642338 for details

This actually represented a much larger change than I initially though, it's been implemented in https://github.com/cefsharp/CefSharp/commit/0e25149c968801be1b945313d9ccbdcd29a7c1f7 and has been, files appear to be correctly copied when using a `Library project` (only one level deep worth of referencing is supported by `msbuild`). 

`Clickonce` should also now be supported as all files are included by `<None/>` entries.

Ideally would move the OffScreen, WinForms and WPF targets file content to the props (as they're actually props now), unfortunately we need guaranteed ordering and Nuget 2.8.x doesn't provide that, if we leave them as .targets it'll happen late enough to make sure the cef.redist list of files is provided.

https://github.com/NuGet/Home/issues/4229#issuecomment-271387190

**Comment by Thicout** [comment]
Hi. I am targeting `AnyCPU`, and since `63.0.0`, I can’t use `Visual Studio Designer` anymore.
I can reproduce the issue with the `MinimalExample` (starting with `55.0.0` for example).

My problem with `63.0.0` (and `63.0.1`) is that, when I target `AnyCPU`, the `CefSharp` references are now `x64`, so the `Visual Studio Designer` doesn’t work. With `57.0.0` the references were `x86` when targeting `AnyCPU`, and I had no issue using the `Visual Studio Designer`.
The build / run of the app is working good with `63.0.0`, but I can’t work with the `Visual Studio Designer`.

I also noticed that after the installation of `63.0.0`, an `AnyCPU` `Project Plateform` is automatically added in the Plateform combobox list in `Visual Studio Configuration Manager`.
It ends with two `AnyCPU` items in the Plateform combobox list, if an `AnyCPU` Plateform was present before the installation of `63.0.0`. And I had strange behavior when I tried to remove the plateform.

If I downgrade to `57.0.0`, everything is working back as expected.
